sortedcontainers/sortedset.pyi

```diff
@@ -0,0 +1,81 @@
+import typing
```

Comment:
This import is only used for `overload`. Can that be added to the `from import` below?